framework/decode/annotation_handler.h

```diff
@@ -40,7 +40,7 @@ class AnnotationHandler
     virtual void ProcessAnnotation(uint64_t               block_index,
                                    format::AnnotationType type,
                                    const std::string&     label,
-                                   const std::string&     data) = 0;
+                                   const std::string&     data){};
```

Comment:
I don't think we want a default NOP implementation. Any class deriving from `AnnotationHandler` should process annotations. I think the original version of the line was right.

Reply:
Done